sdk-python/littlehorse/exceptions.py

```diff
@@ -14,3 +17,11 @@ class SerdeException(Exception):
     def __init__(self, message: str):
         self.message = message
         super().__init__(self.message)
+
+
+class GrpcException(Exception):
```

Comment:
Is there a reason why we don't use the standard grpc exception like we do in the Java SDK?

We wrap it in the server but we don't expose the wrapper to the users in `sdk-java`. The wrapper is just so that it's easier to throw the exception.

Reply:
this is the reason `from grpc._channel import _InactiveRpcError`

`_channel`? `_InactiveRpcError`? i do not like the python grpc lib, do you want me to remove the GrpcException? if so, ther is no problem